catalog/core/catalog-core-commands/src/main/resources/bin/catalog-export.bat

```diff
@@ -45,17 +49,41 @@ if "%FILENAME%"=="" goto USAGE
 if "%KEYSTORE%"=="" goto USAGE
 
 echo Connecting to running DDF and exporting catalog to %FILENAME%...
-call client.bat -h %HOST% "catalog:dump --include-content=%FILENAME% %DIR% "
+
+if "%CQL%"=="" (
+    echo Calling client with command:  catalog:dump --include-content %FILENAME% %DIR%
+    call client.bat -h %HOST% "catalog:dump --include-content %FILENAME% %DIR% "
+) else (
+    echo Calling client with command:  catalog:dump --include-content %FILENAME% --cql ^"%CQL%^" %DIR%
+    call client.bat -h %HOST% "catalog:dump --include-content %FILENAME% --cql ""%CQL%"" %DIR% "
+)
 
 where %JARSIGNER% >nul 2>nul
 if %ERRORLEVEL% NEQ 0 (
-	echo Unable to find %JARSIGNER%, ensure that $JAVA_HOME and $JAVA_HOME/bin are set on the path.
-	exit /b 1
+ echo Unable to find %JARSIGNER%, ensure that $JAVA_HOME and $JAVA_HOME/bin are set on the path.
+ exit /b 1
 ) else (
-	echo Running %JARSIGNER% on %FILENAME%, which will prompt you to enter the keystore password to sign the zip file...
-	call %JARSIGNER% -keystore %KEYSTORE% %DIR%/%FILENAME% %ALIAS% -signedJar %DIR%signed%FILENAME%
+ echo Running %JARSIGNER% on %FILENAME%, which will prompt you to enter the keystore password to sign the zip file...
+ call %JARSIGNER% -keystore %KEYSTORE% %DIR%/%FILENAME% %ALIAS% -signedJar %DIR%signed%FILENAME%
 )
 goto :eof
 
 :USAGE
-echo Usage: ^<-d directory^> ^<-z zipname^> ^<-k keystore location^> [-a alias] [-h hostname]
\ No newline at end of file
+echo Usage: ^<-d directory^> ^<-z zipname^> ^<-k keystore location^> [-a alias] [-h hostname] [-c cql-expr]
```

Comment:
Oversight. Removed the existing `USAGE` statement in favor of the `SYNTAX` to stay consistent with the catalog:dump --help message.